venues/thecvf.com/ECCV/2020/Conference/python/secondary_ACs.py

```diff
@@ -0,0 +1,324 @@
+
+    ac_group = client.get_group('thecvf.com/ECCV/2020/Conference/Area_Chairs')
+    secondary_ac_group = client.post_group(openreview.Group(
+        id = 'thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs',
+        signatures = ['thecvf.com/ECCV/2020/Conference'],
+        signatories = ['thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs'],
+        readers = [
+            'thecvf.com/ECCV/2020/Conference',
+            'thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs'],
+        writers = ['thecvf.com/ECCV/2020/Conference'],
+        members = ac_group.members
+    ))
+    print ('Posted Secondary AC group: {}\n'.format(secondary_ac_group.id))
+
+    aggregate_score_invitation = client.post_invitation(openreview.Invitation(
+        id='thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs/-/Aggregate_Score',
+        readers=[
+            'thecvf.com/ECCV/2020/Conference',
+            'thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs'],
+        writers=['thecvf.com/ECCV/2020/Conference'],
+        invitees=['thecvf.com/ECCV/2020/Conference'],
+        signatures=['thecvf.com/ECCV/2020/Conference'],
+        reply={
+            'readers': {
+                'values-copied': [
+                    'thecvf.com/ECCV/2020/Conference',
+                    '{tail}']
+                },
+            'nonreaders': {
+                'values-regex': 'thecvf.com/ECCV/2020/Conference/Paper.*/Authors'
+                },
+            'writers': {
+                'values': ['thecvf.com/ECCV/2020/Conference']
+                },
+            'signatures': {
+                'values': ['thecvf.com/ECCV/2020/Conference']
+                },
+            'content': {
+                'head': {
+                    'type': 'Note',
+                    'query': {'invitation': 'thecvf.com/ECCV/2020/Conference/-/Blind_Submission'}
+                },
+                'tail': {
+                    'type': 'Profile',
+                    'query': {'group': 'thecvf.com/ECCV/2020/Conference/Secondary_Area_Chairs'}
+                },
+                'weight': {'value-regex': '[-+]?[0-9]*\\.?[0-9]*'},
+                'label': {'value-regex': '.*'}
+            }
+        }))
+    print('Posted aggregate score invitation:', aggregate_score_invitation.id)
+
+    map_submissions = {note.number: note for note in openreview.tools.iterget_notes(client, invitation = 'thecvf.com/ECCV/2020/Conference/-/Blind_Submission')}
+    
+    all_meta_reviews = list(openreview.tools.iterget_notes(client, invitation='thecvf.com/ECCV/2020/Conference/Paper[0-9]*/-/Meta_Review$'))
+    print('Found {} official meta reviews\n'.format(len(all_meta_reviews)))
+    set_reject_papers = set()
+    for meta in all_meta_reviews:
+        paper_num = int(meta.invitation.split('Paper')[1].split('/')[0])
+        if paper_num in map_submissions:
+            if meta.content['recommendation'] == 'Reject':
```

Comment:
instead of reading the recommendation field, you should read the field 'clear_reject' [yes|no], and take in account only the ones with value `no`.